mono/mini/Makefile.am.in

```diff
@@ -34,6 +34,52 @@ sgen_libs = \
 	$(monodir)/mono/utils/libmonoutils.la \
 	$(glib_libs)
 
+if ENABLE_LLVM
```

Comment:
This whole block needs to use tabs indentation like the rest of the file instead of 2-spaces.

Reply:
Commenting with a tab here causes make to interpret these lines as rules

Reply:
I will just omit indentation

Reply:
Correct, ignore this comment then.